editor.planx.uk/src/@planx/components/MapAndLabel/Public/index.tsx

```diff
@@ -76,7 +76,9 @@ const VerticalFeatureTabs: React.FC<{ features: Feature[] }> = ({
               {...a11yProps(i)}
               {...(isFeatureInvalid(i) && {
                 sx: (theme) => ({
-                  backgroundColor: theme.palette.action.focus,
+                  borderRight: `12px solid ${theme.palette.error.main}`,
+                  // Appear over tab indicator
+                  zIndex: 2,
```

Comment:
This is a little hacky as the tabs are buttons which are slightly hidden / obscured by the TabPanel itself (see below).

![image](https://github.com/user-attachments/assets/07475b07-7022-463b-82e7-10f52bffe347)

@ianjon3s Would love another pair of eyes on this if you get a chance!